test/sd59x18/powu/powu.t.sol

```diff
@@ -0,0 +1,215 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.13;
+
+import {
+    E,
+    MAX_SD59x18,
+    MAX_WHOLE_SD59x18,
+    MIN_SD59x18,
+    MIN_WHOLE_SD59x18,
+    PI,
+    PRBMathSD59x18__Exp2InputTooBig,
+    PRBMathSD59x18__LogInputTooSmall,
+    PRBMathSD59x18__PowuOverflow,
+    SD59x18,
+    ZERO,
+    powu
+} from "~/SD59x18.sol";
+import { PRBMath__MulDiv18Overflow } from "~/Helpers.sol";
+import { SD59x18__BaseTest } from "../SD59x18BaseTest.t.sol";
+
+contract SD59x18__PowuTest is SD59x18__BaseTest {
+    SD59x18 internal constant SQRT_MAX_SD59x18 = SD59x18.wrap(240615969168004511545033772477_625056927114980741);
+    SD59x18 internal constant SQRT_MIN_SD59x18 = SD59x18.wrap(-240615969168004511545033772477_625056927114980741);
```

Comment:
This should be renamed to `NEGATIVE_SQRT_MAX_SD59x18`.

`MIN_SD59x18`'s root is a complex number.